src/avs/AVSDetailsTVLTab.jsx

```diff
@@ -0,0 +1,274 @@
+import {
+  Spinner,
+  Table,
+  TableBody,
+  TableCell,
+  TableColumn,
+  TableHeader,
+  TableRow,
+  Image,
+  Skeleton,
+  Spacer,
+  Tooltip
+} from '@nextui-org/react';
+import React, { useMemo, useState } from 'react';
+import {
+  BEACON_STRATEGY,
+  EIGEN_STRATEGY,
+  STRATEGY_ASSET_MAPPING
+} from './helpers';
+import { useTailwindBreakpoint } from '../shared/useTailwindBreakpoint';
+import { formatUSD, formatETH, formatNumber } from '../shared/formatters';
+
+export default function AVSDetailsTVLTab({
+  totalTokens,
+  lst,
+  ethRate,
+  isAVSLoading
+}) {
+  return (
+    <>
+      {/*line chart*/}
+      <div className="bg-content1 border border-outline flex items-center justify-center h-[512px] p-4 rounded-lg w-full">
```

Comment:
I'd rather add bottom margin here for a gap:

```suggestion
      <div className="bg-content1 border border-outline flex items-center justify-center h-[512px] mb-4 p-4 rounded-lg w-full">
```

Reply:
I will fix this in #123 to prevent breaking the graphs